README.md

```
@@ -127,31 +127,7 @@ Supported OS:
-$ sudo dpkg -i packages-microsoft-prod.deb
-$ sudo apt-get update
-$ sudo apt-get install apt-transport-https
-$ sudo apt-get update
-$ sudo apt-get install dotnet-sdk-3.0
-```
-To check installation:
-
-`$ dotnet --version`
-
-will tell you what version of `dotnet` you have.
+#### Supported `.NET 10` 
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)